netplugin/netd.go

```diff
@@ -584,10 +601,9 @@ func main() {
 	//logger := log.New(os.Stdout, "go-etcd: ", log.LstdFlags)
 	//etcd.SetLogger(logger)
 
-	err = handleEvents(netPlugin, crt, opts)
-	if err != nil {
+	if err := handleEvents(netPlugin, crt, opts); err != nil {
 		os.Exit(1)
-	} else {
-		os.Exit(0)
 	}
+
+	select {} // loop
```

Comment:
This `select{}` loop might not be needed now as `handleEvents()` will block until error.